src/assets/release-notes-1.18.1.json

```diff
@@ -0,0 +1,1043 @@
+{
```

Comment:
I think those are too many for the 1.18.1 patch release. I guess we have to ensure that we pass the correct branch on patch releases there:
- https://github.com/kubernetes/release/blob/9a84972e209bf6f885cafe824676164cb4e04090/cmd/krel/cmd/release_notes.go#L722
- https://github.com/kubernetes/release/blob/9a84972e209bf6f885cafe824676164cb4e04090/cmd/krel/cmd/release_notes.go#L764

Reply:
Ok, I'm on it. I'll merge the PR creation stuff and start fixing this one.

Reply:
I've made changes to krel to better calculate te previous tag and to read from the release branch if it exists. This time we got 14 notes from 44 commits (check out the output below).